zookeeper-server/src/test/java/org/apache/zookeeper/common/X509UtilTest.java

```diff
@@ -17,11 +17,30 @@
  */
 package org.apache.zookeeper.common;
 
+import org.apache.zookeeper.PortAssignment;
```

Comment:
nit: revert import reorder? Or fix your import organizer to do java.* -> javax.* -> everything else

Reply:
Yep, no problem. Looks like the import order changed in November last year in these classes.